lib/nfs_common.pm

```diff
@@ -29,6 +29,18 @@ sub server_configure_network {
 sub try_nfsv2 {
     # Try that NFSv2 is disabled by default
     systemctl 'start nfs-server';
+
+    # Make sure it exists and also print the content as info.
+    assert_script_run "cat /proc/fs/nfsd/versions";
```

Comment:
you can consider using a `fail_message` which would also be a way to replace the comment with a text string with function

```suggestion
    assert_script_run 'cat /proc/fs/nfsd/versions', fail_message => 'NFS versions file must exist';
```

Reply:
That's missing the "also print content as info" part

Reply:
well, you get the idea

Reply:
Changed.